src/cloudai/workloads/nemo_run/cloudai_nemorun.py

```diff
@@ -446,6 +446,29 @@ def cloudai_llama3_8b_recipe() -> run.Partial:
 # LLAMA3 70B Recipe
 @run.cli.factory(target=llm.pretrain)
 def cloudai_llama3_70b_recipe() -> run.Partial:
+    nsys_enabled = os.getenv("NSYS", "0") == "1"
```

Comment:
For task and recipe we use `CLOUDAI_...` env vars format, do you think it make sense following this approach for nsys as well? IMO it make sense.